contrib/http_util/src/main/java/io/opencensus/contrib/http/AbstractHttpHandler.java

```diff
@@ -47,7 +50,7 @@
    * @param type the {@code MessageEvent.Type} of the event.
    * @param uncompressedMessageSize size of the message before compressed (optional).
    * @param compressedMessageSize size of the message after compressed (optional).
-   * @since 0.18
+   * @since 0.19
```

Comment:
Please send a separate PR to change all "since" tags that are not released.

Reply:
ok